Makefile

```diff
@@ -46,6 +49,9 @@ iso: nerdctl-$(NERDCTL_VERSION)-$(ARCH) qemu-$(QEMU_VERSION)-copying cri-dockerd
 nerdctl-$(NERDCTL_VERSION)-$(ARCH):
 	curl -o $@ -Ls https://github.com/containerd/nerdctl/releases/download/v$(NERDCTL_VERSION)/nerdctl-full-$(NERDCTL_VERSION)-linux-$(ARCH_ALIAS).tar.gz
 
+openresty-v$(OPENRESTY_VERSION)-$(ARCH):
+	curl -o $@ -Ls https://github.com/rancher-sandbox/openresty-packaging/releases/download/v$(OPENRESTY_VERSION)/$@.tar
```

Comment:
I'm not sure I'm comfortable pulling this in without automated CI (for e.g. when you go on vacation).

Reply:
I'm not happy about it either, but we are running out of time for this release (because the previous release ran over by over a week, and we had to do 2 patch releases). This obviously needs to be done by CI, but I don't have the time right now.